concent_api/core/models.py

```diff
@@ -485,3 +493,478 @@ def clean(self):
             raise ValidationError({
                 'pending_response': 'PaymentInfo should be related with Pending Response'
             })
+
+
+class AbstractMessageModel(Model):
+
+    sig = BinaryField()
+    timestamp = IntegerField()
+    encrypted = BooleanField(default = False)
+
+    class Meta:
+        abstract = True
+
+
+class StoredComputeTaskDef(AbstractMessageModel):
+
+    # Defines items list from ComputeTaskDef
+    ITEMS = [
+        'task_id',
+        'subtask_id',
+        'deadline',
+        'src_code',
+        'extra_data',
+        'short_description',
+        'working_directory',
+        'performance',
+        'docker_images',
+    ]
+
+    task_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH)
+    subtask_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH)
+    deadline = DateTimeField(blank = True, null = True)
+    src_code = TextField()
+    extra_data = JSONField()
+    short_description = TextField()
+    working_directory = TextField()
+    performance = DecimalField(decimal_places=2, max_digits=3)
+    docker_images = JSONField()
+
+    if not len(ITEMS) == len(message.tasks.ComputeTaskDef().ITEMS.keys()):
+        raise ValidationError({
+            'ComputeTaskDef': 'StoredComputeTaskDef model has not same fields as golem_messages.message.tasks.ComputeTaskDef()'
+        })
```

Comment:
This validation starts when we start django server. If any message would change their structure in golem_message repo we will know it from beginning. It can be also assert and as I saw below we can do it with `set() == set()`

Reply:
So actually it should be function containing assert(s), I think, as this validation is repeated in every class, it should be extracted to function.